content/cpp/concepts/math-functions/terms/exp/exp.md

```diff
@@ -1,55 +1,56 @@
 ---
 Title: 'exp()'
-Description: 'Returns e raised to the power of the argument.'
+Description: 'Returns the base-e exponential function of the argument.'
 Subjects:
   - 'Computer Science'
 Tags:
   - 'Functions'
   - 'Arithmetic'
+  - 'Methods'
 CatalogContent:
   - 'learn-c-plus-plus'
   - 'paths/computer-science'
 ---
 
-`exp()` returns the base-_e_ exponential function of the argument, which is _e_ raised to the power of the argument.
-
-The number _e_, also known as Euler's number, is a mathematical constant approximately equal to 2.71828 and the base of the natural logarithm.
-
-The `cmath` library must be added to the top of the file with `#include <cmath>`.
+The **exp()** function returns the base-_e_ exponential function of the argument. This is _e_ raised to the power of the argument. This is the inverse of the [`log()`](https://www.codecademy.com/resources/docs/cpp/math-functions/log) function.
 
 ## Syntax
 
-Returns *e*ⁿ:
-
 ```cpp
-std::exp(n)
+ exp(x)
 ```
 
+The `x` parameter can be any value, positive, negative or zero.
+
 ## Example
 
-Use `exp()` to return the value of *e*³):
+The following example uses the `exp()` function to find the exponential function of `3`:
```

Comment:
```suggestion
The following example uses the `exp()` function to find the value of e<sup>3</sup>:
```

Reply:
I will use this suggestion for the section devoted to the Codebyte example.

```diff
-The following example is runnable and returns the exponential function of `1`:
+The following example is runnable and returns the value of e<sup>1</sup>:
```